public/Anirudh Anup

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Anirudh Anup",
+  "bio": "I am Anirudh Anup, A passionate 13 year old developer from India, I am always open to collaborating on projects",
+  "avatar": "http://github.com/qwe123coder.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "http://github.com/qwe123coder",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on polywork",
+      "url": "https://www.polywork.com/anirudh_anup",
+      "icon": "polywork"
```

Comment:
I don't think polywork is support yet, so this will get filter out for now and not display

Reply:
oh ok, so should I remove it?